docs/site/decorators/Decorators_inject.md

```diff
@@ -178,6 +178,39 @@ using `binding.to(value)`.
 To set other types of value providers such as `toDynamicValue`or `toClass`, use
 `@inject.binding` instead.
 
+### @instance
+
+`@instance` injects an instance of a given class from a binding created using
+`toClass()`.
+
+Syntax:
+`@instance(serviceClass?: Constructor, metadata: {skipSubClasses?: boolean})`.
+
+```ts
+class MyController {
+  constructor(@instance(MyService) public myService: MyService) {}
```

Comment:
What are the pros and cons of `@instance(MyService)` when compared to something like `@inject.instanceOf(MyService)`? Personally, I slightly prefer the longer & more descriptive variant.

@strongloop/loopback-next thoughts?

Reply:
Let's go with `@inject.instance` then ✅